glusterd2/xlator/options/options.go

```diff
@@ -71,6 +71,21 @@ const (
 	OptionStatusDeprecated
 )
 
+func (l OptionLevel) String() string {
+	switch l {
+	case OptionStatusBasic:
```

Comment:
Question: Do you really want the API to return response values starting with caps ? `Basic` or `basic` ? JSON responses are usually all small caps. Let me know what you think.

Reply:
I just copy pasted the auto-generated code. 
we have lot of String functions which returns string with Capital letters in gd2 and same as sent as response to restclient.

example from volume response

 ``` "type": "Distributed-Replicate"```
 ```"type": "Brick"```

Reply:
Fair enough :+1: